pkg/kubelet/config/flags.go

```diff
@@ -95,8 +95,8 @@ func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 
 	// Network plugin settings for Docker.
 	fs.StringVar(&s.NetworkPluginName, "network-plugin", s.NetworkPluginName, fmt.Sprintf("<Warning: Alpha feature> The name of the network plugin to be invoked for various events in kubelet/pod lifecycle. %s", dockerOnlyWarning))
```

Comment:
curious if this is still alpha? this flag is here for a long time, but docs saying 1.19: https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/

Reply:
Good catch, going to remove the alpha warning here too.

I've always wondered why the CNI flags are still marked "alpha". My assumption is that we added them initially and we just forgot to remove them.

Reply:
We should also update the docs if we remove it here